src/samples/UWP/Uno.Themes.Samples.Mobile/Uno.Themes.Samples.Mobile.csproj

```diff
@@ -0,0 +1,101 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<TargetFrameworks>net7.0-ios;net7.0-android;net7.0-maccatalyst</TargetFrameworks>
+		<SingleProject>true</SingleProject>
+		<OutputType>Exe</OutputType>
+		<!-- Display name -->
+		<ApplicationTitle>Uno.Themes.Samples</ApplicationTitle>
+		<!-- App Identifier -->
+		<ApplicationId>com.companyname.Uno.Themes.Samples</ApplicationId>
```

Comment:
We'll need to align with what xamarin targets had: https://github.com/unoplatform/Uno.Themes/blob/23fa1ad75b42ea78fa14f48832907f324ce376bf/src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Droid/Properties/AndroidManifest.xml#L2